assets/css/style.css

```diff
@@ -392,3 +392,104 @@ main {
 .hidden {
   display: none;
 }
+
+.contact-section {
+  width: 100%;
+  background-image: url('../images/backgrounds/contact-me.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+}
+
+.contact-background {
+  padding: 2em 2em 0 2em;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 5em 0;
+  margin-bottom: 0;
+  background-position: right 15px;
+  background-repeat: no-repeat;
+  max-width: 30em;
+}
+
+.contact-form > h2 {
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+.contact-form-paragraph {
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  margin-bottom: 2em;
+}
+
+.contact-form > .input-group > input,
+.contact-form > .input-group > textarea {
+  /* Mono/White */
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin-bottom: 1em;
+  padding: 0.5em;
+  width: 100%;
```

Comment:
Please make sure your design is the same as the given Figma design. You should stick to the Figma design according to the project requirements.

![Screenshot from 2023-06-08 17-59-19](https://github.com/chudisoft/Portfolio-Microverse/assets/101523582/a3b8c684-d361-4167-b12b-668b833c1d23)

Your design
![image](https://github.com/chudisoft/Portfolio-Microverse/assets/101523582/e1e8a4b5-078a-4c61-a28e-7d8dc4e6ed00)

Figma design
![image](https://github.com/chudisoft/Portfolio-Microverse/assets/101523582/fd30efa7-4260-4735-ab92-a34a0641a311)

Kindly, check all the `color`,  `font-type`, `font-weight`, `margin` and `height` from the Figma design